frontend/src/components/FullWidthAlert.tsx

```diff
@@ -25,7 +25,7 @@ const getBGColor = (type: Props["type"]) => {
 
 const FullWidthAlert = ({ type, heading, children }: Props) => {
   return (
-    <div className={`${getBGColor(type)}`}>
+    <div className={`${getBGColor(type)} tablet:position-sticky top-0 z-top`}>
```

Comment:
The background color is computed, based on the type of alert — e.g. blue if an info alert, red if an error. The others are hard-coded for all use cases.

Reply:
Oh, the reason we have to do this — @SammySteiner and I found it was the simplest solution — is because we're putting a regular ol' alert (which has a max-width) inside a full-width container that has a matching background color that'll extend the color to the device edges.